manifests/charts/istio-cni/values.yaml

```diff
@@ -40,7 +40,7 @@ cni:
   chained: true
 
   # Allow the istio-cni container to run in privileged mode, needed for some platforms (e.g. OpenShift)
-  privileged: false
+  privileged: true
```

Comment:
Looks like this may increase privileges for non-ambient users?

Reply:
Even in sidecar mode, the CNI container is literally copying binaries to root-accessible system directories as a matter of function - it has always effectively required node-level root perms to do that.

That's what the this section in the CNI daemonset does:

```
          securityContext:
            runAsGroup: 0
            runAsUser: 0
            runAsNonRoot: false
            privileged: {{ .Values.cni.privileged }}
```

we are always running as root on the node whether the privileged flag is set or not. OpenShift just demands you be more explicit about it :D

Reply:
Makes sense to me, thanks!